src/VisualStudio/Xaml/Impl/Implementation/LanguageClient/XamlInProcLanguageClient.cs

```diff
@@ -23,8 +23,14 @@ internal class XamlInProcLanguageClient : AbstractInProcLanguageClient
     {
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, true)]
-        public XamlInProcLanguageClient(XamlLanguageServerProtocol languageServerProtocol, VisualStudioWorkspace workspace, IDiagnosticService diagnosticService, IAsynchronousOperationListenerProvider listenerProvider, XamlSolutionProvider solutionProvider)
-            : base(languageServerProtocol, workspace, diagnosticService, listenerProvider, solutionProvider, diagnosticsClientName: null)
+        public XamlInProcLanguageClient(
+            XamlLanguageServerProtocol languageServerProtocol,
+            VisualStudioWorkspace workspace,
+            IDiagnosticService diagnosticService,
+            IAsynchronousOperationListenerProvider listenerProvider,
+            XamlSolutionProvider solutionProvider,
+            DefaultCapabilitiesProvider defaultCapabilitiesProvider)
```

Comment:
the xaml provider needs to specify a different set of capbilities - see their handler here 
http://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices.Xaml/Implementation/LanguageServer/Handler/Initialize/InitializeHandler.cs,9663db7ec51dd43e